contracts/handlers/GenericHandler.sol

```diff
@@ -28,10 +28,10 @@ contract GenericHandler is IGenericHandler {
     mapping (address => bytes32) public _contractAddressToResourceID;
 
     // contract address => deposit function signature
-    mapping (address => bytes4) public _contractAddressToDepositFunctionSignature;
+    mapping (address => bytes4) public _contractAddressToDepositFunctionSignature; // REVIEW: Can be moved into _resourceIDToContractAddress. Will save 21000 gas on setResource.
```

Comment:
I think packing the function signaturs together into `_contractAddressToResourceID` might make sense since everything is some set length.